### PR TITLE
Photometry

### PR DESCRIPTION
**Pull request by dannygoldstein**
Addresses #187

### CODE REVIEW

**Comment by acrellin** [comment]
@dannygoldstein looks like this PR and #182 have some overlap -- what's the status of these? Should we close and try to get the other in? thanks!

File: skyportal/models.py [hunk 1]
```diff
@@ -239,9 +245,18 @@ class Photometry(Base):
     mjd = sa.Column(sa.Float)  # mjd date
     time_format = sa.Column(sa.String, default='iso')
     time_scale = sa.Column(sa.String, default='utc')
-    mag = sa.Column(sa.Float)
-    e_mag = sa.Column(sa.Float)
+
```
Comment (dannygoldstein) [inline skyportal/models.py]: I think the only things that need to be merged here are the change to this file, but I would also keep mag + e_mag as fields.

Comment (dannygoldstein) [reply skyportal/models.py]: ZTF alerts do not provide flux/fluxerr, but offline forced photometry does. Best to have all of these as fields, and require either mag or flux, but not both.

Danny

**Comment by acrellin** [comment]
Closing, as we're not going with q3c, and moving updates to photometry data model to https://github.com/skyportal/skyportal/pull/430

**Pull request closed without merging**